app/src/molecules/MoleculeAlert.re

```diff
@@ -0,0 +1,105 @@
+open Atom;
+open Particle;
+
+type severity =
```

Comment:
Uncertain about this and `t` below as alluded to in another comment down below. My initial reaction was that it should be in a place unrelated to the presentation layer, but then again it's used in here mostly.

Reply:
I moved the `alert` type to the store, but will leave the `severity` type here, as it's mostly used in this module.

Reply:
See comment in the store.